utilities/test_suite/rpp_test_suite_common.h

```diff
@@ -1045,8 +1044,8 @@ inline void compare_reduction_output(Rpp32f* output, string funcName, RpptDescPt
     refFile = refPath + "/../REFERENCE_OUTPUT/" + funcName + "/"+ func + ".csv";
 
     ifstream file(refFile);
-    Rpp32f *refOutput;
-    refOutput = (Rpp32f *)malloc(srcDescPtr->n * 4 * sizeof(Rpp32f));
+    Rpp64u *refOutput;
+    refOutput = (Rpp64u *)malloc(srcDescPtr->n * 4 * sizeof(Rpp64u));
```

Comment:
We've always used calloc for all instances, lets  do the same here to ease future searches.